### PR TITLE
add dollar value in Token

### PR DESCRIPTION
**Pull request by Thykof**
image

### CODE REVIEW

**Comment by Thykof** [comment]
@pivilartisant you can continue this PR if you have the designs

**Comment by pivilartisant** [comment]
I've update the pr in accordance with designs, i am waiting on designer confirmation of the implementation

File: src/components/Token/Token.tsx [hunk 1]
```diff
@@ -25,18 +26,35 @@ export function Token(props: TokenProps) {
...
+    const { amountFormattedPreview, amountFormattedFull } = formatFTAmount(
+      bigintBalance,
+      decimals,
+    );
+    rawBalance = amountFormattedFull;
+    formattedBalance = amountFormattedPreview;
+  } else {
+    formattedBalance = undefined;
+    rawBalance = undefined;
+  }
```
Comment (Thykof) [inline src/components/Token/Token.tsx:50]: let's initialize the variables to underfined line 35 37

**Review by pivilartisant — APPROVED** [review]

**Comment by Thykof** [comment]
Let's add a prop that we will use like this:

```
dollarValueError={
              token.dollarValue === undefined || token.dollarValue === ''
                ? Intl.t('errors.dollar-value')
                : undefined
            }
```

**Comment by Thykof** [comment]
https://github.com/massalabs/ui-kit/pull/422

**Pull request merged**